nsqd/http.go

```diff
@@ -596,3 +600,41 @@ func (s *httpServer) printStats(stats []TopicStats, health string, startTime tim
 	}
 	return buf.Bytes()
 }
+
+type allowedOpts struct {
+	NSQLookupdTCPAddresses []string `json:"nsqlookupd_tcp_addresses"`
```

Comment:
It may be nice to support these as atomic add/remove _or_ overwrite. The way I've done this in the past, which isn't _too_ onerous, is

```go

type allowedOpts struct {
    NSQLookupdTCPAddresses []string `json:"nsqlookupd_tcp_addresses"`
    NSQLookupdTCPAddressesAdd []string `json:"nsqlookupd_tcp_addresses_add"`
    NSQLookupdTCPAddressesRemove []string `json:"nsqlookupd_tcp_addresses_remove"`
}
```

That way I have the option of overwriting the values _completely_, or I can just add a new address and remove a different one.